src/VisualStudio/IntegrationTest/TestUtilities/InProcess/DialogHelpers.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Automation;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess
+{
+    public static class DialogHelpers
+    {
+        public static AutomationElement FindDialog(int visualStudioHWnd, string dialogAutomationName, bool isOpen)
```

Comment:
I still hate this API where we're using this to both handle "find me this open dialog" and "wait for it to go away and I'll return null because sure why not."